considerations.mkd

```diff
@@ -58,3 +58,38 @@ precision than necessary.
    represents edges as great circles will have different interpretations
    of edges, which vary more the longer the edges are. Try to
    avoid edges of more than 180 degrees in length as far as possible.
+
+
+# Extensibility
+
+In GeoJSON, an "extension" is any property not defined in section 2.
+Consuming implementations that encounter unfamiliar extensions MUST NOT
+stop processing or signal an error and MUST continue processing the
+items as if those properties were not present. Note that support for
+extensions can vary across implementations and no normative processing
+model for extensions is defined. Accordingly, implementations that rely
+too heavily on the use of extensions may experience reduced
+interoperability with other implementations.
+
+## GeoJSON types are fixed
+
+Implementations MUST NOT extend the fixed set of GeoJSON types:
+FeatureCollection, Feature, Point, LineString, MultiPoint, Polygon,
+MultiLineString, MultiPolygon, and GeometryCollection.
+
+## Semantics of GeoJSON properties are fixed
+
+Implementations MUST NOT change the the semantics of GeoJSON properties.
+
```

Comment:
if we are not going to allow extensions such as `"is_a_circle_not_a_multipoint"`, would there be consideration for adding `properties` or another private namespace inside of a GeoJSON types?

wrapping something in a Feature in order to add a single element can be very verbose.

the (contrived) case I am thinking of is:

```
{
  "type": "Point",
  "coordinates": [ 0, 0, 100 ],
  "moment": true
}
```

in this case, it would be encapsulating of WKT `POINT M (1 1 100)`.